index.html

```diff
@@ -49,121 +49,203 @@ <h2>My Recent Works</h2>
                 </ul>
-                <button class="recent-button" type="button">see project</button>
+            <button class="project-button" type="button">See Project</button>
+            </div>
+            <div class="container">
+                <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project-tools">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            <button class="project-button" type="button">See Project</button>
+            </div>
+            <div class="container">
+                <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project-tools">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            <button class="project-button" type="button">See Project</button>
+            </div>
+            <div class="container">
+                <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project-tools">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            <button class="project-button" type="button">See Project</button>
+            </div>
+            <div class="container">
+                <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project-tools">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            <button class="project-button" type="button">See Project</button>
+            </div>
+            <div class="container">
+                <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project-tools">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            <button class="project-button" type="button">See Project</button>
```

Comment:
- [ ] Kindly check the video above to have a clearer picture of how your portfolio is rendering at the moment. The behaviour is not so smooth for the following reasons:
  - [ ] No background image is showing
  - [ ] No project description can be seen or is readable on the page
  - [ ] Projects section is not responsive
  - [ ] The hovering effects that allows some content of the project card to be hidden while the `see project` action pops up has not been implemented yet or not visible.
  - [ ] The style properties are hardcoded with absolute units (e.g. `px`) causing overflows, and some elements are too big - the project cards such that screen sizes above `960px` can not contain a `3` column grid to match design.

This is what gets rendered :point_down: 
![image](https://user-images.githubusercontent.com/99715640/222934404-465bc0d1-6467-4931-9246-b89c42d5e431.png)